cosmic-applet-time/src/config.rs

```diff
@@ -0,0 +1,20 @@
+use cosmic::cosmic_config::cosmic_config_derive::CosmicConfigEntry;
+use cosmic::cosmic_config::{self, CosmicConfigEntry};
+
+#[derive(Debug, Clone, CosmicConfigEntry, PartialEq, Eq)]
+#[version = 1]
+pub struct TimeAppletConfig {
+    pub military_time: bool,
```

Comment:
nit: we should probably call this 24-hour time, as that is what it's referred to more often.

Reply:
It was already called that [in cosmic-settings](https://github.com/pop-os/cosmic-settings/blob/353cf3fbd120a365fb08aed973fed3ea29cad41c/cosmic-settings/src/pages/time/date.rs#L19), but we can change it.

Reply:
Ah, I should bring the conversation up there then. No need to change anything here!